src/app/component/general/eco-news/news-list/news-list-list-view/news-list-list-view.component.html

```diff
@@ -0,0 +1,30 @@
+<div id="list-gallery-content">
+  <div class="list-image">
+    <img class="list-image-content" src="../../../../../../assets/img/tamplate-image-list.png" alt="user added image" />
+  </div>
+  <div class="added-data">
+    <div class="filter-tag">
+      <p class="ul-eco-buttons">{{ ecoNewsModel.tag }}</p>
+    </div>
+    <div class="title-list">
+      <p>
+        {{ ecoNewsModel.title | slice: 0:70 }}
+      </p>
+    </div>
+    <div class="list-text">
+      <p>
+        {{ ecoNewsModel.text | slice: 0:200 }}
+      </p>
+    </div>
+    <div class="user-data-added-news">
+      <p class="user-data-text-date-share">
+        <img class="icon" [src]="profileIcons.userIcon" alt="user icon" />{{ ecoNewsModel.name }}
+      </p>
+      <p class="user-data-text-date-share">
+        <img class="icon" [src]="profileIcons.calendarIcon" alt="calendar icon" />{{ ecoNewsModel.creationDate }}
```

Comment:
If you have more than 2 attributes, please replace it to one column

Reply:
Fixed